### ISSUE TITLE
this[method] is not a function

### ISSUE THREAD

**Issue by jmikedupont2** [dzervoudakes/react-scaffolder#7]
```
msomething@DESKTOP-K34VJ99:~/GitHub/mcc/generators$ npm install -g generator-react-scaffolder

added 108 packages, and audited 109 packages in 7s

12 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
msomething@DESKTOP-K34VJ99:~/GitHub/mcc/generators$ yo react-scaffolder --ts
? What is the name of the application? test
? Yarn? (Y/n) y
   create test/.DS_Store
   create test/.editorconfig
   create test/.eslintignore
   create test/.eslintrc.js
   create test/.lintstagedrc.js
   create test/.npmrc
   create test/.postcssrc.js
   create test/.prettierignore
   create test/.prettierrc.js
   create test/.stylelintignore
   create test/.stylelintrc.js
   create test/jest.config.js
   create test/tsconfig.json
   create test/typedoc.json
   create test/webpack.config.js
   create test/build/webpack.base.js
   create test/build/webpack.dev.js
   create test/build/webpack.prod.js
   create test/public/favicon.ico
   create test/scripts/build.ts
   create test/scripts/eslint.ts
   create test/scripts/stylelint.ts
   create test/scripts/test.ts
   create test/src/App.tsx
   create test/src/globals.d.ts
   create test/src/index.tsx
   create test/src/jest-dom.d.ts
   create test/src/polyfills.ts
   create test/src/__tests__/App.test.tsx
   create test/src/routes/Routes.tsx
   create test/src/routes/index.ts
   create test/src/scss/_constants.scss
   create test/src/scss/_default.scss
   create test/src/scss/style.scss
   create test/src/components/Footer/Footer.scss
   create test/src/components/Footer/Footer.tsx
   create test/src/components/Footer/index.ts
   create test/src/components/Header/Header.scss
   create test/src/components/Header/Header.tsx
   create test/src/components/Header/index.ts
   create test/src/components/Layout/Layout.scss
   create test/src/components/Layout/Layout.tsx
   create test/src/components/Layout/index.ts
   create test/src/components/SectionHeader/SectionHeader.scss
   create test/src/components/SectionHeader/SectionHeader.tsx
   create test/src/components/SectionHeader/index.ts
   create test/src/pages/Home/Home.tsx
   create test/src/pages/Home/index.ts
   create test/src/pages/Missing/Missing.tsx
   create test/src/pages/Missing/index.ts
   create test/src/routes/__tests__/Routes.test.tsx
   create test/src/components/Footer/__tests__/Footer.test.tsx
   create test/src/components/Header/__tests__/Header.test.tsx
   create test/src/components/Layout/__tests__/Layout.test.tsx
   create test/src/components/SectionHeader/__tests__/SectionHeader.test.tsx
   create test/src/pages/Home/__tests__/Home.test.tsx
   create test/src/pages/Missing/__tests__/Missing.test.tsx
   create test/.gitignore
   create test/package.json
   create test/README.md
   create test/public/index.html

Installing dependencies with yarn...

Error react-scaffolder --ts

this[method] is not a function
```

**Comment by dzervoudakes** [comment]
@jmikedupont2 Apologies for the delay here.  I missed a breaking change in v5 of `yeoman`.  Will have this addressed soon.

**Comment by dzervoudakes** [comment]
This has been addressed and released in [v1.9.0](https://github.com/dzervoudakes/react-scaffolder/releases/tag/v1.9.0), which also includes some dependency management for `npm 7.x` (should still be backwards compatible with `npm 6.x`).  Yarn compatibility is unchanged.

Thank you for catching this issue.
